packages/vscoq-language-server/vscoq-language-server.2.2.6/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Enrico Tassi" "Maxime Dénès" "Romain Tetley" ]
+license: "MIT"
+homepage: "https://github.com/coq-community/vscoq"
+bug-reports: "https://github.com/coq-community/vscoq/issues"
+dev-repo: "git+https://github.com/coq-community/vscoq"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" { >= "4.14" }
+  "dune" { >= "3.5" }
+  "coq-core" { ((>= "8.18" < "8.21") | (= "dev")) }
+  "coq-stdlib" { ((>= "8.18" < "8.21") | (= "dev")) }
```

Comment:
```suggestion
  "coq-core" { ((>= "8.18" < "9.1") | (= "dev")) }
  "coq-stdlib" { ((>= "8.18" < "9.1") | (= "dev")) }
```

Reply:
@ivnhk I don't have the rights to push this commit and re-run CI, please do.
@rtetley please upstream this change.